src/Shell.hs

```diff
@@ -63,6 +94,11 @@ doInterpret (GenericCmd "exit" (code:_)) = case TR.decimal code of
   (Left str) -> do
     liftIO $ TIO.putStrLn $ "Error: wrong exit code: " `T.append` code
     return []
+doInterpret (DeclCmd var expr) = do
+  val <- evalExpr expr
+  setVar var val
+  liftIO $ putStrLn (show var ++ show val)
```

Comment:
Not the prettiest way to display this - do we even want to show this by default?